lektor/admin/static/webpack.config.js

```diff
@@ -1,8 +1,9 @@
 var webpack = require('webpack')
 var path = require('path')
```

Comment:
Feel free to update these to const as well.

Reply:
Will do